extensions.json

```diff
@@ -885,6 +885,12 @@
       "repository": "https://github.com/michaelgmcd/vscode-language-babel",
       "version": "0.0.30"
     },
+    {
+      "id": "miguel-savignano.ruby-symbols",
+      "repository": "https://github.com/MiguelSavignano/vscode-ruby-symbols",
+      "version": "0.1.8",
+      "checkout": "master"
```

Comment:
Unfortunately, the author forgot to tag the 0.1.8 version with a `v0.1.8` tag, and we can't really use `master` for this.

So, let's publish straight from the default branch, without pinning a specific version (that's the best we can do here.)